### PR TITLE
Reduced csaber stamina damage

### PR DESCRIPTION
**Pull request by PrimeNumb**
## About the PR

It changes the `attack()` proc for `/obj/item/sword/`.
The `stamina_damage` var that is inherent to items is modified when activated/deactivated. Previously the csaber would apply both its default stamina damage (by calling its parent `attack()` proc), and also a whopping 205 stamina damage through the `do_disorient()` proc.

## Why's this needed?

This patch is intended to slightly reduce the stamina damage dealt by the csaber to make it more on par with the katana in terms of usefulness (both being 7 TC items).
In practice the patch will make it so the first hit will disorient the target, and the second hit will stun (assuming no armor and no stamina boosters are involved). 
For people wearing riot armor it'll be about 2-3 hits for a stun (depending on stamina ticks).

Previously the csaber would instantly stun any target without stamina modifiers, which was kind of powerful for a 7 TC item. The brute damage is the same, but this'll give people an actual chance to disarm instead of being onetapped out of nowhere. The csaber has a fair amount of stamina cost, so the chance to disarm (especially when being attacked by multiple people) is not insignificant.

## Changelog

RichardGere:
* Reduced the stamina damage dealt by csabers

### CODE REVIEW

File: code/obj/item/misc_weapons.dm [hunk 1]
```diff
@@ -46,11 +46,13 @@
...
 	contraband = 5
 	desc = "An illegal weapon that, when activated, uses cyalume to create an extremely dangerous saber. Can be concealed when deactivated."
-	stamina_damage = 35
+	stamina_damage = 35 // This gets applied by obj/item/attack, regardless of if the saber is active.
 	stamina_cost = 30
 	stamina_crit_chance = 35
 	var/do_stun = 1 //controlled by itemspecial for csword. sorry.
 	var/active_force = 60
+	var/active_stamina_dmg = 150
+	var/inactive_stamina_dmg = 35
```
[the remark this answers is not in the document]
Comment (ZeWaka) [reply code/obj/item/misc_weapons.dm:55]: Nah, keep it.

**Comment by geneva-breaker** [comment]
This is sorely needed and a step in the right direction- we should get a few comments from players in here.

**Comment by Flaborized** [comment]
Good change imo

**Comment by Sartorius7** [comment]
Yes please. The csaber's ability to singlehandedly decimate the majority of the crew needs a little nerf.

**Comment by ZeWaka** [comment]
The katana always seemed like a weak csaber that had no purpose to buy, this evens the cards a bit.

Also, instastuns suck

**Comment by Tarmunora** [comment]
Good change. If it turns out to be too significant a nerf, consider adding a bit of proj and/or disarm resist to the sabre while active.

**Pull request merged**